src/olympia/users/tests/test_commands.py

```diff
@@ -75,39 +76,54 @@ def test_adds_supergroup(self):
 
 
 class TestClearOldUserData(TestCase):
+    def create_ip_log(self, user):
+        # Note: this is a dummy log that doesn't have store_ip=True on
+        # purpose, to ensure we directly use IPLog when it comes to
+        # deletion - so even if we change our minds about storing ips for
+        # a given action after a while, we'll still correctly clear the old
+        # data when it's time to do so.
+        activity = ActivityLog.create(amo.LOG.CUSTOM_TEXT, 'hi', user=user)
+        IPLog.objects.create(activity_log=activity, ip_address='127.0.0.56')
```

Comment:
Nit: Making the `ip_address` a const would avoid having to hardcode it in so many places in the tests below.

Reply:
Yeah, I wanted to follow the style of the tests I was modifying which does repeat the value.